docs/source/installation.rst

```diff
@@ -16,8 +16,10 @@ npm is the easiest way to install docxtemplater
 
     npm install docxtemplater
     npm install jszip@2
+    npm install jszip-utils
```

Comment:
Can you add this comment : 

 
+    npm install jszip-utils # for browser support (load docx via HTTP)